cms-adapter/src/Rsbc.Dmf.CaseManagement.Service/Services/DocumentService.cs

```diff
@@ -119,6 +119,88 @@ public async override Task<GetDmerReply> GetDmer(CaseIdRequest request, ServerCa
                 result.Item.Status = dmerCase.DmerStatus ?? string.Empty;
                 result.Item.Provider = new Provider();
                 result.Item.Provider.Name = dmerCase.Login?.FullName ?? string.Empty;
+                result.Item.DocumentId = dmerCase.DocumentId.ToString();
+                result.ResultStatus = ResultStatus.Success;
+                
+            }
+            catch (Exception ex)
+            {
+                result.ResultStatus = ResultStatus.Fail;
+                result.ErrorDetail = ex.Message;
+            }
+
+            return result;
+        }
+
+        // TODO driver id is no longer relevant and method name should be changed
+        public async override Task<GetDriverAndCaseDocumentsReply> GetDriverAndCaseDocuments(GetDriverAndCaseDocumentsRequest request, ServerCallContext context)
```

Comment:
this method was removed

Reply:
I think i have merged that in will remove that method